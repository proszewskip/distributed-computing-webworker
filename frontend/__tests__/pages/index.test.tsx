import React from 'react';
import renderer from 'react-test-renderer';

import Index from '../../pages/index';

describe('index page', () => {
  it('should match snapshot', () => {
    const component = renderer.create(<Index isAuthenticated={true} />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
