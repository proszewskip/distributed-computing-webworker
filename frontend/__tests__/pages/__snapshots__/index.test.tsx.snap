// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index page should match snapshot 1`] = `null`;
